src/app/pages/fund/components/skip-fund-account-button.tsx

```diff
@@ -0,0 +1,26 @@
+import { Button } from '@stacks/ui';
+import { FundPageSelectors } from '@tests/page-objects/fund.selectors';
+
+interface SkipFundAccountButton {
+  onSkipFundAccount(): void;
+}
+export function SkipFundAccountButton({ onSkipFundAccount }: SkipFundAccountButton) {
+  return (
+    <Button
+      _focus={{ boxShadow: 'none' }}
+      _hover={{
+        boxShadow: 'none',
+      }}
```

Comment:
```suggestion
      _hover={{ boxShadow: 'none' }}
```

nit, consistency with line above

Reply:
Merged this, so will fix on my open PR bc minor.